integration-test/473-landuse-tier.py

```diff
@@ -13,8 +13,7 @@
 # area 30089300
 assert_has_feature(
     8, 56, 115, 'landuse',
-    { 'kind': 'national_park', 'id': -921675, 'tier': 1,
-      'min_zoom': 8 })
+    { 'kind': 'national_park', 'tier': 1, 'min_zoom': 8 })
 assert_no_matching_feature(
     7, 28, 57, 'landuse',
     { 'kind': 'national_park', 'id': -921675 })
```

Comment:
Should we take the whole `assert_no_matching_feature` test out, then? Or leave it in, but it doesn't seem like it's very useful as a test any more. :wink: